agent/custom/bbc_connection_manager.py

```
@@ -397,19 +486,16 @@ def _launch_bbc(self):
-                    mfaalog.info(f"[BbcConnectionManager] BBC 就绪事件: {event}")
-                    return True
+        """等待 BBC 就绪信号"""
+        mfaalog.info(f"[BbcConnectionManager] 等待BBC就绪 (超时{timeout}s)...")
+        ready = self._bbc_ready_event.wait(timeout=timeout)
         
-        mfaalog.warning(f"[BbcConnectionManager] 等待 BBC 就绪超时 ({timeout}s)")
-        return False
+        if ready:
+            mfaalog.info("[BbcConnectionManager] BBC 就绪事件已触发")
+            return True
+        else:
```

Ruff (0.15.10)

[warning] Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)